packages/aggregate-client/src/aggregate.js

```diff
@@ -50,12 +44,12 @@ export async function aggregateOffer(
  * Get details of an aggregate.
  *
  * @param {import('./types').InvocationConfig} conf - Configuration
- * @param {import('@ucanto/interface').Link<unknown, number, number, 0 | 1>} commitmentProof
+ * @param {import('@ucanto/interface').Link<unknown, number, number, 0 | 1>} subject
```

Comment:
```suggestion
 * @param {import('@ucanto/interface').UnknownLink} subject
```

There is an `UnknownLink` type that covers version CID 0 as well.